project-geometry/README.metadata.json

```
@@ -0,0 +1,34 @@
+{
+    "category": "Geometry",
```

```suggestion
    "category": "Edit and Manage Data",
```
This should resolve the GitHub checker warning.